model/firewalls_private_subnets.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require_relative "../model"
+
+class FirewallsPrivateSubnets < Sequel::Model
```

Comment:
Do we need this pivot model?

Reply:
Unfortunately yes, because we cannot use association_dependencies to handle the join table destroy.